tests/test_check_stac_metadata.py

```diff
@@ -694,6 +698,72 @@ def should_successfully_validate_partially_uploaded_dataset(subtests: SubTests)
                     )
 
 
+@mark.timeout(timedelta(minutes=20).total_seconds())
+@mark.infrastructure
+def should_mark_asset_as_replaced_in_new_version() -> None:
```

Comment:
There are two tests with the exact same name. Based on #1779 I assume that this test was not meant to be here.